profiler/src/ProfilerEngine/Datadog.Profiler.Native.Linux/LinuxStackFramesCollector.cpp

```diff
@@ -388,7 +383,7 @@ void LinuxStackFramesCollector::CollectStackSampleSignalHandler(int signal, sigi
             if (pCollectorInstance->CanCollect(OpSysTools::GetThreadId(), info->si_pid))
             {
                 // In case it's the thread we want to sample, just get its callstack
-                auto resultErrorCode = pCollectorInstance->CollectCallStackCurrentThread();
+                auto resultErrorCode = pCollectorInstance->CollectCallStackCurrentThread(reinterpret_cast<unw_context_t*>(context));
```

Comment:
You don't need to cast it here since the method expects a `void*`
```suggestion
                auto resultErrorCode = pCollectorInstance->CollectCallStackCurrentThread(context);
```

Reply:
✔️